.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: php
 
-php: 5.6
+php: 7.3
```

Comment:
Might be worth adding `dist: xenial` here to be explicit and consistent with other modules?